src/main/java/appeng/items/tools/quartz/ToolQuartzWrench.java

```diff
@@ -103,14 +103,13 @@ public boolean isUsable( ItemStack item, EntityLivingBase user, Entity entity )
 	@Override
 	public void toolUsed( ItemStack item, EntityLivingBase user, BlockPos pos )
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1186)